.hooks/check_categories-list.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+categories_list=("ai" "architecture" "cloud" "data" "dev" "general" "green" "mobile" "ops" "web")
```

Comment:
Can you externalize this list into a `config.json` in order to reuse it anywhere like